src/anndata/_core/aligned_mapping.py

```diff
@@ -396,9 +415,39 @@ def __init__(
     ):
         self.parent_mapping = parent_mapping
         self._parent = parent_view
-        self.subset_idx = (subset_idx, subset_idx)
+        self.subset_idx = subset_idx
```

Comment:
@ivirshup I think this was you.  Why?

Reply:
Ah ok, it's because of this https://github.com/scverse/anndata/pull/1119/files#diff-30201eb006d4117775a934d53f6f3474312b98528a978c41220ffa18a3360fb9R456-R457